internal/dashboard/business/ac/enforcers.go

```diff
@@ -58,3 +58,26 @@ func PathGuard(domain, resource, action string) EnforceFunc {
 		return true, nil
 	}
 }
+
+func unionHelper(allPass bool, enforces ...EnforceFunc) EnforceFunc {
```

Comment:
I think it can be more straight forward to use an enum value indicating 'and' or 'or'

Reply:
OK, but the `allPass` var is still required for simplicity.